src/views/Swap/styles.tsx

```diff
@@ -4,8 +4,11 @@ import styled from 'styled-components'
 export const StyledSwapContainer = styled(Flex)<{ $isChartExpanded: boolean }>`
   flex-shrink: 0;
   height: fit-content;
-  ${({ $isChartExpanded }) => ($isChartExpanded ? 'padding: 0 40px' : 'padding: 0 40px')};
+  ${({ $isChartExpanded }) => ($isChartExpanded ? 'padding: 0 24px' : 'padding: 0 24px')};
 
+  ${({ theme }) => theme.mediaQueries.lg} {
+    ${({ $isChartExpanded }) => ($isChartExpanded ? 'padding: 0 40px' : 'padding: 0 40px')};
+  }
```

Comment:
lol
I just noticed that the stuff on the left and the right side of ternaries here are the same 😂 
I guess let's just remove the expanded prop. Looks good anyway.

Reply:
updated lol